src/core.c/Distribution/Locally.pm6

```diff
@@ -1,4 +1,4 @@
-role Distribution::Locally does Distribution {
+role Distribution::Locally does Distribution::Checksum {
```

Comment:
`Distribution::Locally` doesn't require a checksum method

Reply:
Initially, I was lacking information about the exact usages of each kind of `Distribution::` and wasn't sure if there is no way for any of them to end up in `$*DISTRIBUTION`. 

But later on, with wider picture in mind, I got to the conclusion since distribution meta data, generally speaking, affects actions taken upon the distribution itself, there must be a way to know if meta has changed. This consideration makes me think that it has to be a fundamental property of the distribution. In a way `meta` itself can serve the purpose with one huge _but_ when it comes to the current implementation: it is not guaranteed to reflect the original meta, as provided by the source. In particular, `InstallationDistribution` is served with `ver` and `api` fields replaced with `Version` objects. `LazyDistribution` reports a subset of keys. `Distribution::Path` adds `files` key.

Therefore the only way for CURD to get the real checksum of the original data is for these implementations to provide it computed at some point _before_ that change anything.

So, CURD could be the only one to have the method but only when we guarantee that the meta provided is what the low-level object has be initialized with. I'd only be happy if this principle is implemented.